R/textmodel_wordfish.R

```diff
@@ -55,17 +60,19 @@
 #' @note In the rare situation where a warning message of "The algorithm did not
 #'   converge." shows up, removing some documents may work.
 #' @seealso [predict.textmodel_wordfish()]
-#' @references Slapin, J. & Proksch, S.O. (2008).
```

Comment:
We should add a description of the new return objects containing the word-level SEs and new parameters.